public/utils/helpers.js

```diff
@@ -46,3 +47,27 @@ export const inputLimitText = (
     </EuiText>
   );
 };
+
+export async function getAlerts({ params, httpClient, notifications, location, history }) {
+  /* const params = {
+    from,
+    size,
+    search,
+    sortField,
+    sortDirection,
+    severityLevel,
+    alertState,
+    monitorIds
+  }; */
```

Comment:
Dev comments like these can likely be removed unless you'd like to use the comment to explain a piece of code that might be tough to understand.